go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/elastic/elastic-package
 
-go 1.21.0
+go 1.22.0
```

Comment:
shouldn't this be `1.22.2`?

Reply:
This is like the lower version supported by the package, not really the version used for compilation.